components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/AbstractAPIManager.java

```diff
@@ -808,9 +808,17 @@ public String getDocumentationContent(APIIdentifier identifier, String documenta
                              documentationName;
         String tenantDomain = MultitenantUtils.getTenantDomain(APIUtil.replaceEmailDomainBack(identifier.getProviderName()));
         Registry registry;
+
+        boolean isTenantFlowStarted = false;
         try {
+            if (tenantDomain != null && !MultitenantConstants.SUPER_TENANT_DOMAIN_NAME.equals(tenantDomain)) {
+                isTenantFlowStarted = true;
```

Comment:
Fixed in #1494(https://github.com/wso2/carbon-apimgt/commit/4320fe8699ada60d3e9727d1389d3febd433999a)